src/hotspot/share/runtime/globals.hpp

```diff
@@ -2094,7 +2094,10 @@ const intx ObjectAlignmentInBytes = 8;
       "replaces the initializing VM on success")                            \
                                                                             \
   product(ccstr, CREngine, "criuengine", "Path or name of a program "       \
-      "implementing checkpoint/restore")                                    \
+      "implementing checkpoint/restore and optional extra parameters "      \
+      "as a comma-separated list of key[=value] pairs; "                    \
+      "-XX:CREngine=program,key=value,anotherkey results in calling "       \
+      "'program --key value --anotherkey'")                                 \
```

Comment:
There is CRAC_CRIU_OPTS that is used to pass additional options to CRIU. I see that remains in place, and this turns out to be alternative?

Not every CREngine may want to follow this convention of argument passing. Usually, `program,--key,value,--anotherkey` is specified to call `program --key value --anotherkey`.

This probably needs to be RESTORE_SETTABLE.